### PR TITLE
[stable/mysql] Add prometheus metrics

### PR DESCRIPTION
**Pull request by xasor**

### CODE REVIEW

**Comment by andrewhowdencom** [comment]
Oh dude yes thanks!

(I am not a project just excited about this PR)

**Comment by xasor** [comment]
;)

File: stable/mysql/templates/svc.yaml [hunk 1]
```diff
@@ -7,6 +7,11 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.metrics.enabled }}
+  annotations:
```
Comment (unguiculus) [inline stable/mysql/templates/svc.yaml:11]: Don't hard-code the annotations. Just make them configurable. You don't need them if you use the prometheus-operator.

File: stable/mysql/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.4.4
+version: 0.4.5
```
Comment (unguiculus) [inline stable/mysql/Chart.yaml]: Should be a minor bump.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
cc @gianrubio

**Comment by xasor** [comment]
Thanks for the review @unguiculus

**Review by unguiculus — APPROVED** [review]

**Pull request merged**